test/k8s-integration/main.go

```diff
@@ -342,6 +358,30 @@ func handle() error {
 		}()
 	}
 
+	// For windows cluster, it has both Windows nodes and Linux nodes. Before triggering the tests, taint Linux nodes
+	// with NoSchedule to avoid test pods being scheduled on Linux. Need to do this step after driver is deployed.
```

Comment:
We need to do this after so that the driver part that runs on Linux won't be evicted? Can we add a toleration to the driver instead?

Reply:
actually we do have tolerations https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/deploy/kubernetes/base/node_linux/node.yaml#L103
but there is some issue when I try to taint linux node before deploying it. Let me double check.

Reply:
I found it, the controller does not have the tolerations. I think we want to avoid setting tolerations to controller to make sure it can be scheduled to a "good" node.

Reply:
In general, the driver should be more resilient to ordering, but I guess this is a test-only scenario where by default we don't want pods to be scheduled to linux nodes. 

In a normal scenario, is setting taints the recommended way? Or do users need to explicitly set nodeSelectors for the OS node they want?

Reply:
I think nodeSelector is better for its flexibility. Taint windows nodes when starting a cluster to avoid some system pods scheduled to Windows ( I guess those Pods do not have node selector). After cluster is up, user should remove windows node taint, then schedule pods based on node selector.